### PR TITLE
Enhanced code docu

### PR DESCRIPTION
**Pull request by ThorstenBux**

### CODE REVIEW

File: AndroidStudioProjects/ARMovieProj/aRMovie/src/main/java/org/artoolkit/ar/samples/ARMovie/ARMovieApplication.java [hunk 1]
```diff
@@ -83,7 +83,7 @@ protected void initializeInstance() {
```
Comment (BoiseBaked) [inline AndroidStudioProjects/ARMovieProj/aRMovie/src/main/java/org/artoolkit/ar/samples/ARMovie/ARMovieApplication.java]: Annoying tab versus 4 spaces.

File: AndroidStudioProjects/ARNativeES1Proj/aRNativeES1/src/main/java/org/artoolkit/ar/samples/ARNativeES1/ARNativeES1Application.java [hunk 2]
```diff
@@ -83,7 +83,7 @@ protected void initializeInstance() {
     	
 		// Unpack assets to cache directory so native library can read them.
```
Comment (BoiseBaked) [inline AndroidStudioProjects/ARNativeES1Proj/aRNativeES1/src/main/java/org/artoolkit/ar/samples/ARNativeES1/ARNativeES1Application.java]: tab versus 4 spaces chars.

File: AndroidStudioProjects/ARNativeOSGProj/aRNativeOSG/src/main/java/org/artoolkit/ar/samples/ARNativeOSG/ARNativeOSGApplication.java [hunk 3]
```diff
@@ -83,7 +83,7 @@ protected void initializeInstance() {
```
Comment (BoiseBaked) [inline AndroidStudioProjects/ARNativeOSGProj/aRNativeOSG/src/main/java/org/artoolkit/ar/samples/ARNativeOSG/ARNativeOSGApplication.java]: tabs versus 4 space char

File: AndroidStudioProjects/calib_opticalProj/calib_optical/src/main/java/org/artoolkit/ar/utils/calib_optical/calib_optical_Application.java [hunk 4]
```diff
@@ -72,7 +72,7 @@ protected void initializeInstance() {
```
Comment (BoiseBaked) [inline AndroidStudioProjects/calib_opticalProj/calib_optical/src/main/java/org/artoolkit/ar/utils/calib_optical/calib_optical_Application.java]: tabs versus four space chars

File: AndroidStudioProjects/nftBookProj/nftBook/src/main/java/org/artoolkit/ar/samples/nftBook/nftBookApplication.java [hunk 5]
```diff
@@ -83,7 +83,7 @@ protected void initializeInstance() {
```
Comment (BoiseBaked) [inline AndroidStudioProjects/nftBookProj/nftBook/src/main/java/org/artoolkit/ar/samples/nftBook/nftBookApplication.java]: tabs versus 4 space chars

File: AndroidStudioProjects/nftSimpleProj/nftSimple/src/main/java/org/artoolkit/ar/samples/nftSimple/nftSimpleApplication.java [hunk 6]
```diff
@@ -83,7 +83,7 @@ protected void initializeInstance() {
```
Comment (BoiseBaked) [inline AndroidStudioProjects/nftSimpleProj/nftSimple/src/main/java/org/artoolkit/ar/samples/nftSimple/nftSimpleApplication.java]: tabs versus 4 space chars

**Comment by BoiseBaked** [comment]
This is ready to merge. Let me know if you want to skip the "tabs versus 4 space chars" issue and I'll click the PR merge button.

**Comment by ThorstenBux** [comment]
I went through the related projects and let the code formatter run on all java classes.
That replaced the tab with spaces and rearranged the code as well. 

Please have a look and merge.

Thanks
—





Thorsten Bux | Developer Advocate
thorsten.bux_c@daqri.com <mailto:thorsten.bux_c@daqri.com>




www.artoolkit.org

**Comment by philip-lamb** [comment]
If this is just formatting and comments, I am happy for it to be merged. John to push the button.

**Pull request merged**